controllers/networkplugins_controller.go

```diff
@@ -54,7 +54,7 @@ const (
 	OvsImage                = "docker.io/platform9/openvswitch:v2.12.0"
 	OvsCniImage             = "quay.io/kubevirt/ovs-cni-plugin:v0.16.2"
 	OvsMarkerImage          = "quay.io/kubevirt/ovs-cni-marker:v0.16.2"
-	HostPlumberImage        = "docker.io/platform9/hostplumber:v0.3"
+	HostPlumberImage        = "docker.io/platform9/hostplumber:v0.3-pmk-2632584"
 	KubeRbacProxyImage      = "gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0"
```

Comment:
@ShaunakJoshi1407  The `KubeRbacProxyImage` will need an update too in luigi (in a similar PR), right?

Reply:
Sure, I'll make the change here

Reply:
Thanks @ShaunakJoshi1407